### ISSUE TITLE
[Bookdown request] A textbook for learning stats using easystats

### ISSUE THREAD

**Issue by CWen001** [easystats/blog#43]
Thanks for all the wonderful packages in easystats. 

I understood that there are already many excellent documentation and blog posts, but is it planned to have a complete textbook to learn basic concepts in stats and the uses of easystats in a systematic way? 

For example, [moderndive](https://moderndive.com/) is a very good bookdown to learn intro level stats and also the package `infer`. [Spatial Data Science](https://keen-swartz-3146c4.netlify.com/) is recommended to learn spatial analysis using their `sf` and `stars`. Similarly, as the philosophy of easystats, it aims to make R stats easy by creating many new and intuitive tools. Maybe it is good to have a stats-newbie-friendly book to guide step by step and to plot the big picture of all these tools in helping learn stats easier.

**Comment by DominiqueMakowski** [comment]
@CWen001 Thank you for the encouragement and this wonderful suggestion.

While it would definitely be great to work and get such a big piece out, I'm afraid that it's not in the plans for now 😞 

One of the reasons is that the ecosystem is not yet fully stable yet, and major work is still done on some parts (although we're getting there!). So we prefer to focus on having the ecosystem in perfect shape first.

That being said, the main reason is likely the lack of resources. Such a project (writing a book) requires countless hours of work, and this is something none of us can afford for now, as all of the contributors are already doing that as a side unpaid job. We already struggle to write as many blogposts (https://github.com/easystats/blog/issues/14) and documentation examples as we would like.

Long story short, I wish we had the manpower, or funding, or time, to do that, but currently we have none 😅

**Comment by CWen001** [comment]
Thank you very much for your response, and I truly understand. Hope `easystats` can get more resources and support.  As a user, I will keep trying this ecosystem and see what I can contribute in the future. Closing this issue for now.

**Comment by DominiqueMakowski** [comment]
@CWen001 thanks for your support!
